docs/GUIDE.md

```diff
@@ -0,0 +1,123 @@
+# Step-by-step guide for Akka Serialization Helper usage
+This document is a guide on how to use Akka Serialization Helper in your project. If you want to get a more general view of this toolbox, see [README](../README.md).
+
+Akka Serialization Helper (ASH) consists of two major parts:
+1. Circe Akka Serializer
+2. Compiler Plugins
```

Comment:
And also sbt plugin right?

Reply:
Hm, yes. I would check this sentence to:

Akka Serialization Helper (ASH) has two major parts that can be used:
1. Circe Akka Serializer
2. Compiler Plugins

Would it be ok? I just wanted to create a view of our toolbox from the user's perspective

Reply:
Okay... let it be so